profiles/arch/amd64/package.use.mask

```diff
@@ -372,11 +372,6 @@ dev-vcs/git -mediawiki
 # Boost.Context can be built on amd64
 dev-libs/boost -context
 
-# Michał Górny <mgorny@gentoo.org> (2013-12-15)
```

Comment:
Could you fix the commit message to clearly indicate it's about new-dynarec? The package "supported amd64" already.

Reply:
Is the updated message okay?